loky/backend/context.py

```diff
@@ -113,6 +118,11 @@ def cpu_count():
        set by docker and similar container orchestration systems);
      * the value of the LOKY_MAX_CPU_COUNT environment variable if defined.
     and is given as the minimum of these constraints.
+
+    If ``maybe_physical_only`` is True, return the number of physical cores
+    unless it can't be found or the number of processors available for the
+    current process has been set by one of the above.
```

Comment:
I would rather rename to `only_physical_cores=False` while explaining more about the limitations in the docstring:

If `only_physical_cores` is `True`, return the number of physical cores instead of the number
logical cores (hyperthreading / SMT). Note that this option is not enforced if the number of
usable cores is controlled in any other way such as: process affinity, restricting CFS scheduler
policy or the LOKY_MAX_CPU_COUNT environment variable.

Reply:
okay, I was sure you would not like the previous name :)